README.md

```diff
@@ -47,6 +47,7 @@ Configuring Faucet application via environment variable:
 - `FAUCET_ACCOUNT_SECRET_KEY` The account that faucet aepp will top off the account. (Required)
 - `TOPUP_AMOUNT` The amount of tokens that the faucet application will place into your account. (Default: 5AE)
 - `SPEND_TX_PAYLOAD` Value to use to fill the payload for the transactions (Default: `Faucet Tx`)
+- `SPEND_TX_TTL` Spend transaction TTL. Higher value needs to be used in fast beating networks i.e. Hyperchains (Default: `3`)
```

Comment:
```suggestion
- `SPEND_TX_TTL` Spend transaction relative TTL. Higher value needs to be used in fast beating networks i.e. Hyperchains (Default: `3`)
```
TTL by itself is usually relative TTL + current height. `SPEND_TX_TTL` maybe obvious, but I got confused.